Add converter docs (#212)
larq/compute-engine: docs/converter.md

# LCE Converter

> LCE allows to convert a Keras model built with `larq` to the TensorFlow Lite FlatBuffer format for inference.

Reviewer: ```suggestion
LCE converter allows to convert a `Larq's` Keras model to an LCE-compatible TensorFlow Lite FlatBuffer format for inference.
```

Reviewer 2: I would personally stick with "Keras model built with larq" . I like the "LCE-compatible" part.

```suggestion
The LCE converter allows you to convert a Keras model built with `larq` to an LCE-compatible TensorFlow Lite FlatBuffer format for inference.
```
